package.json

```
@@ -11,7 +11,7 @@
     "start:dev": "node docker/scripts/reloader.js --inspect=0.0.0.0:9229 -r ts-node/register docker/scripts/start-kuzzle-dev.ts --enable-plugins functional-test-plugin",
     "prepublishOnly": "npm run build",
     "build": "npm run build-ts",
-    "clean": "npm run build | grep TSFILE | cut -d' ' -f 2 | xargs rm",
+    "clean": "touch index.ts && npm run build | grep TSFILE | cut -d' ' -f 2 | xargs rm",
```

A better and cleaner way to do that:
```suggestion
    "clean-ts": "git clean -fdX -e node_modules",
    "clean": "git clean -fdX",
```

---

I don't like this command because it removes EVERY untracked files from the working dir :expressionless:  
I just loss some temp working files

---

When it comes to removing files I want to know exactly what I am removing